Bump of Spotbugs to 4.1.3 breaks reporting

**Describe the bug**
While working on issue 2806 I was running a full build including site generation to make sure that everything works OK even for the release-case.

This results in an error for the Spotbugs plugin:
```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Dependency-Check 6.0.3-SNAPSHOT:
[INFO] 
[INFO] Dependency-Check ................................... SUCCESS [ 16.278 s]
[INFO] Dependency-Check Utils ............................. FAILURE [ 19.316 s]
[INFO] Dependency-Check Core .............................. SKIPPED
[INFO] Dependency-Check Command Line ...................... SKIPPED
[INFO] Dependency-Check Ant Task .......................... SKIPPED
[INFO] Dependency-Check Maven Plugin ...................... SKIPPED
[INFO] Dependency-Check Plugin Archetype .................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  35.974 s
[INFO] Finished at: 2020-10-01T22:00:16+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Cannot create file-based resource. /Users/aikebah/NetBeansProjects/DependencyCheck/utils/target/file_/Users/aikebah/NetBeansProjects/DependencyCheck/utils/../src/main/config/spotbugs_excludes.xml (No such file or directory) -> [Help 1]
```

Downgrading spotbugs back to 4.0.4 resolves the problem and makes the build pass successfully.

Looks to me as if the new version of spotbugs can no longer handle the `..` in a spotbugs settings configuration URL anymore.

**Version of dependency-check used**
Build of the main branch HEAD revision

**Log file**
Full log for mvn -s settings.xml install site site:stage -DreleaseTesting -Danalyzer.node.package.enabled=false -Danalyzer.node.audit.enabled=false -e -X can be found in a [gist](https://gist.github.com/aikebah/7f380219dd0325badc52f3166c3fd502)

**To Reproduce**
Steps to reproduce the behavior:
1. Clone the repository
2. Run `mvn -s settings.xml install site site:stage -DreleaseTesting -Danalyzer.node.package.enabled=false -Danalyzer.node.audit.enabled=false` and see it fail
3. Run `mvn -s settings.xml install site site:stage -DreleaseTesting -Danalyzer.node.package.enabled=false -Danalyzer.node.audit.enabled=false  -Dspotbugs.maven.plugin.version=4.0.4` and see it succeed

--- a/pom.xml
+++ b/pom.xml
@@ -370,7 +370,7 @@ Copyright (c) 2012 - Jeremy Long
                                 if ("dependency-check-parent".equals("${project.artifactId}")) {
                                     config = "file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/src/main/config"
                                 } else {
-                                    config = "file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/../src/main/config"
+                                    config = new URI("file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/../src/main/config").normalize().toString()
                                 }
                                 project.properties['odc.config']= config
                                 ]]></script>
@@ -389,7 +389,7 @@ Copyright (c) 2012 - Jeremy Long
                                 if ("dependency-check-parent".equals("${project.artifactId}")) {
                                     config = "file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/src/main/config"
                                 } else {
-                                    config = "file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/../src/main/config"
+                                    config = new URI("file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/../src/main/config").normalize().toString()
                                 }
                                 project.properties['odc.config']= config
                                 ]]></script>
@@ -409,7 +409,7 @@ Copyright (c) 2012 - Jeremy Long
                                 if ("dependency-check-parent".equals("${project.artifactId}")) {
                                     config = "file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/src/main/config"
                                 } else {
-                                    config = "file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/../src/main/config"
+                                    config = new URI("file:///"+project.basedir.absolutePath.replace('\\','\\\\')+"/../src/main/config").normalize().toString()
                                 }
                                 project.properties['odc.config']= config
                                 ]]></script>

